interface/modules/custom_modules/oe-module-dorn/public/orders.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once "../../../../globals.php";
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Modules\Dorn\ConnectorApi;
+use OpenEMR\Core\Header;
+//this is needed along with setupHeader() to get the pop up to appear
+
+$tab = "orders";
+$pageTitle = xl("DORN Orders");
+if (!AclMain::aclCheckCore('admin', 'users')) {
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => $pageTitle]);
+    exit;
+}
```

Comment:
acl?

Reply:
I believe orders and results for user access is available in openemr lab structure same as always. Brad needs to verify but non-admin user don't need access to all the DORN setup and management UI. Might be best to manage from menu level permissions. @claimrevolution can you verify for us?